src/tile/tile.scss

```diff
@@ -127,7 +122,7 @@
           background-color: rgba(0, 0, 0, t(iui-opacity-4));
         }
 
-        > svg {
+        > .iui-icon {
```

Comment:
Seems we were applying iui-icon before, so this should not infuence anything, hopefully..